lib/public/Group/Backend/ABackend.php

```diff
@@ -65,4 +66,33 @@ public function implementsActions($actions): bool {
 
 		return (bool)($actions & $implements);
 	}
+
+	/**
+	 * @since 25.0.0
+	 */
+	public function groupsExists(array $gids): array {
+		$existingGroups = [];
+		foreach ($gids as $gid) {
+			$exists = $this->groupExists($gid);
+			if ($exists) {
+				$existringGroups[] = $gid;
```

Comment:
```suggestion
				$existingGroups[] = $gid;
```
How come psalm does not catch this typo?